### PR TITLE
Proofread javascript/eslint.md

### LINKED ISSUE #1279: The irony issue:  Proofread Eslint and Linting outlines
Task:

- [ ] Proofread these two files for spelling and markdown:
  - javascript/eslint.md
  - linting/code-linting-eslint.md 
- [ ] Complete @vegetabill's requests made in this review: https://github.com/Techtonica/curriculum/pull/1244/files
- [ ] Change the titles of the two files to be more relevant - ie not exactly the same!

### PR DESCRIPTION
**Pull request by Adyasha8105**
1. Proofread these two files for spelling and markdown:
     javascript/eslint.md
     linting/code-linting-eslint.md
     (And these two files are merged into one file.)
2. Complete @vegetabill's requests made in this review: 
    (https://github.com/Techtonica/curriculum/pull/1244/files)
3. Change the titles of the two files to be more relevant.

I have done all these given tasks.
Resolves #1279.

### CODE REVIEW

**Comment by Adyasha8105** [comment]
@alodahl please check my PR.

**Comment by Adyasha8105** [comment]
@alodahl This file doesn't exist as the stuff of that was moved to javascript/eslint.md

**Comment by alodahl** [comment]
Please fix both files, wherever they live now.

**Comment by Adyasha8105** [comment]
Hey @alodahl, I understand your concern. But, the files have been completely changed now. linting/code-linting-eslint.md does not exist. See this commit (https://github.com/Techtonica/curriculum/commit/eb77873bc9dc69410007ceeb58a0599c03983753#diff-6a1ba4c9a11230881bdf211dcc5a4753). And the file which exists now that is javascript/eslint.md, I have made the necessary changes asked in issue, but I can't change the requirements for the non-existing file i.e linting/code-linting-eslint.md. Lemme know what do you think.

**Comment by Adyasha8105** [comment]
@alodahl  Can you tell me what do you think?

File: javascript/eslint.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+- Guided Practice: 5-10 min (Mock Project)
+- Independent Practice: 30min
 - Check for Understanding: 1 min
 
 ### Prerquisites
 
 Here are topics you must be comfortable with to get the maximum benefit out of this topic:
 
--  [Javascript 6 - Object Literals](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-6-object-literals.md)
+- [Javascript 6-Object Literals](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-6-object-literals.md)
```
Comment (alodahl) [inline javascript/eslint.md]: I'm not sure about this change.  Check out the md preview 
image

File: javascript/eslint.md [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+# Linting with ESLint Manual
 
 ### Projected Time
 60 minutes
 
 - Lesson: 15-20 min
-- Guided Practice: 5 -10 min (Mock Project)
-- Independed Practice: 30min
+- Guided Practice: 5-10 min (Mock Project)
+- Independent Practice: 30min
```
Comment (alodahl) [inline javascript/eslint.md]:
```suggestion
- Independent Practice: 30 min
```

If we're being consistent.

File: javascript/eslint.md (hunk 1 above)
Comment (alodahl) [reply javascript/eslint.md]: Oh also, the word "Prerquisites" needs help

**Comment by Adyasha8105** [comment]
Thanks for merging @alodahl :) .

**Comment by Kundan28** [comment]
Nice work @Adyasha8105 🎉 
Updated it in scoresheet too.

**Comment by Adyasha8105** [comment]
> Nice work @Adyasha8105 🎉
> Updated it in scoresheet too.

Thanks 👍 , @Kundan28.

**Comment by Adyasha8105** [comment]
@Kundan28 @alodahl 
(https://user-images.githubusercontent.com/54095539/77859792-fe5d3100-7228-11ea-953c-7e25f1bf80a7.png)
Here my email ID is written wrong and it should be adyashamohanty8105@gmail.com.
Can you change it?

**Comment by Kundan28** [comment]
@Adyasha8105 That's not added manually by somebody it's due to your configuration settings I think so.

**Comment by Adyasha8105** [comment]
@Kundan28 Do you know how to change it?

**Comment by Kundan28** [comment]
> @Kundan28 Do you know how to change it?

No, I searched for it but didn't get any result.

**Comment by Adyasha8105** [comment]
> > @Kundan28 Do you know how to change it?
> 
> No, I searched for it but didn't get any result.

@Kundan28 okay :(

**Pull request merged**